recipes/dask/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: dask
+  version: "{{ version }}"
+
+source:
+  fn: dask-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
+  md5: f111b837ae969a690cf23cd75add92e2
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
Please change to `python setup.py install --single-version-externally-managed --record=record.txt`.